data/mods/ssb/moves.ts

```diff
@@ -112,6 +112,47 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Normal",
 	},
 
+	// Kaiju Bunny
+	cozycuddle: {
+		accuracy: 95,
+		basePower: 0,
+		category: "Status",
+		desc: "Traps the target and lowers its Attack and Defense by two stages.",
+		shortDesc: "Traps target. Atk -2 Def -2.",
+		name: "Cozy Cuddle",
+		pp: 20,
+		priority: 0,
+		flags: {},
+		volatileStatus: 'cozycuddle',
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onTryHit(target, source, move) {
+			if (target.volatiles['cozycuddle']) return false;
+			if (target.volatiles['trapped']) {
+				delete move.volatileStatus;
+			}
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Flatter', target);
+			this.add('-anim', source, 'Let\'s Snuggle Forever', target);
+		},
+		onHit(target, source, move) {
+			this.boost({atk: -2, def: -2}, target, target);
+		},
+		effect: {
+			onStart(pokemon, source) {
+				this.add('-start', pokemon, 'move: Cozy Cuddle', '[of]' + source);
```

Comment:
shouldn't this be `source.name`

Reply:
I'll change it, but that's how it was in Octolock